package.json

```diff
@@ -19,6 +19,6 @@
     "code-push": "^1.1.1-beta"
   },
   "devDependencies": {
-    "react-native": "0.11.4"
+    "react-native": "0.14.2"
```

Comment:
Do we need to update this to 0.15.0? Or does this version support everything we need?